src/index.html

```diff
@@ -7,8 +7,90 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <a href="#home">
+        <img
+        src="images/logo.png"
+        class="logo"
+        alt="home">
+      </a>
+
+      <nav class="navigation">
+        <ul class="navigation__list">
+          <li class="list__element">
+            <a href="#apple"
+            class="nav__link is-active"
+            >
+              apple
+            </a>
+          </li>
+
+          <li class="list__element">
+            <a href="#samsung"
+            class="nav__link"
+            >
+              samsung
+            </a>
+          </li>
+
+          <li class="list__element">
+            <a href="#smartphones"
+            class="nav__link">
+              smartphones
+            </a>
+          </li>
+
+          <li
+          class="list__element"
+          >
+            <a href="#laptops&computers"
+            class="nav__link"
+            data-qa="hover"
+            >
+              laptops & computers
+            </a>
+          </li>
+
+          <li class="list__element">
+            <a href="#gadgets"
+            class="nav__link"
+            >
+              gadgets
+            </a>
+          </li>
+
+          <li class="list__element">
+            <a href="#tablets"
+            class="nav__link"
+            >
+              tablets
+            </a>
+          </li>
+
+          <li class="list__element">
+            <a href="#photo"
+            class="nav__link"
+            >
+              photo
+            </a>
+          </li>
+
+          <li class="list__element">
+            <a href="#video"
+            class="nav__link nav__link--last"
+            >
+              video
+            </a>
+          </li>
+        </ul>
+      </nav>
+
+    </header>
```

Comment:
There shouldn't be any empty lines between child and parent, remove empty line between header and nav and header and body

Reply:
Empty lines 102, 104 are not necessary, remove them